apps/web/pages/checkout.vue

```diff
@@ -184,7 +142,6 @@ const { shippingPrivacyAgreement, setShippingPrivacyAgreement } = useAdditionalI
 const i18n = useI18n();
 const paypalCardDialog = ref(false);
 const termsAccepted = ref(false);
```

Comment:
Has to be removed, we rename the `checkboxValue` to `termsAccepted` by doing `checkboxValue`: `termsAccepted`. So that `termsAccepted` is used again.